### PR TITLE
Update readme.md_Pallavi_Singla

### PR DESCRIPTION
**Pull request by psinglap**
## Objective

_Note why you're making this change and any relevant issues_

## Changes made

_Note what you actual changes you're proposing_ 

## Who needs to review this Pull Request? 

_@vikasgargbear pls approve
_Teammates, leave comments in a review if you approve this change_

### CODE REVIEW

**Comment by psinglap** [comment]
@joewadcan

File: 2024_Spring/readme.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Make the Pull Request against changing the file in _my repo_ (not your fork). I
 
 - [ ] Singh, Manas
 
-- [ ] Singla, Pallavi
+- [x] Singla, Pallavi
 
 - [ ] Soni, Anupam
```
Comment (psinglap) [inline 2024_Spring/readme.md:86]: Amazing

**Comment by jainaman93** [comment]
@psinglap : please add your profile name

**Comment by psinglap** [comment]
Pallavi Singla

**Review by psinglap — COMMENTED** [review]
It is done

**Comment by jainaman93** [comment]
@psinglap : ask for approval

**Comment by psinglap** [comment]
@vikasgargbear pls approve :P :P

**Review by vikasgargbear — COMMENTED** [review]
approved

**Pull request merged**